Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /app
 COPY . .
 
 RUN npm install --production
+RUN npm i --save-dev @types/bcrypt
```

Comment:
Also not great form but I didn't have any luck with the `tsc` command without this line. Tried adding the lib in as a dev/normal dependency but heroku didn't like that. Suggestions welcome if someone knows a better way to do this

Reply:
I added `typescript` as a dependency (previously dev dependency) in my most recent PR, not sure if you had set this stuff up before that change was on main. If not, give it another go because _I think_ that fixes it, but if not this is fine. Just lmk

Reply:
@keithradford Assuming you're talking about [this PR](https://github.com/SENG499-Company-4/Backend/pull/38/files), I did have that merged into this branch and still needed this line

Reply:
Ok, no worries. We can try and find a fix later but doesn't seem like a huge deal.